### DISCUSSION TITLE
Murex could use a logo

### DISCUSSION THREAD

**Discussion by lmorg** [lmorg/murex#632] [Announcements]
This is perhaps the toughest issue for me to resolve because I'm not even remotely any good at graphic design

**Comment by ben-alkov** [comment]
https://en.wikipedia.org/wiki/Murex - there's a lovely photo of a Murex pecten. Someone with the right skillset could use an [autotracer](https://en.wikipedia.org/wiki/Image_tracing) to create an SVG...

**Comment by kmatt** [comment]
The characters "µx", Mu and X, make for a simple logo. In Helvetica for example. "User experience."

I could not find a good font containing a character for "µRx".

**Comment by ben-alkov** [comment]
> The characters "µx", Mu and X, make for a simple logo

+1 That's really clever

**Comment by lmorg** [comment]
A quick 5 minute play on https://www.fontspace.com/symbola-font-f22021 produced this:
![text-1663752379700](https://user-images.githubusercontent.com/12153701/191468894-d42fe9c8-9b87-4d60-acf5-024bb3892d7b.png)

I'm not in love with it but it's definitely far better than my past efforts, thus far.

(the shell is the unicode symbol U+1F41A (Spiral Shell) https://unicode-table.com/en/search/?q=shell)

**Comment by ben-alkov** [comment]
Fun fact: I use '🐚' as my shell prompt (not my joke - someone else gifted me that one years ago :D )

**Comment by lmorg** [comment]
> Fun fact: I use '🐚' as my shell prompt (not my joke - someone else gifted me that one years ago :D )

I'm not sure if that's genius or insane, but I'm going to steal that idea and do the same

**Comment by kmatt** [comment]
This is a suggestion for a simple project logo, where @ben-alkov suggestion of an actual shell as a prompt is more fun.


I used Helvetica because it was handy, and I'm not overly creative when it comes to font choices:

![image](https://user-images.githubusercontent.com/435936/191790538-f25ea81d-a76f-4113-85ba-640276401887.png)

My terminal font of choice is Fira Code which looks like this:

![image](https://user-images.githubusercontent.com/435936/191792858-4147e7b5-3a34-4046-bc3a-9fb6c6c21a60.png)

https://www.fontspace.com/megastructure-font-f85375 - my non-designer attempt to use color, loosely based on the background of murex.rocks:

![image](https://user-images.githubusercontent.com/435936/191824288-0e4874e1-a2f1-4c3b-983b-58621a33cb48.png)
 
![image](https://user-images.githubusercontent.com/435936/191826739-c97f957f-310f-41d3-8599-e3c7e2768d9b.png)

**Comment by lmorg** [comment]
I've had another play and have come up with the following

![murex-logo-2](https://github.com/lmorg/murex/assets/12153701/8afb1c73-553b-4d2e-a7be-04cc2340aae1)

I can't say I'm 100% happy with this but it does feel like the best thus far. Maybe if I drop the "MUREX" lettering and maybe go with a different sea shell (rather than sticking with one that is technically accurate)?

**Comment by yamcodes** [comment]
I'm a big fan of the work so I thought I'd show my support by helping with the logo hunt.

I noted the current keywords and colors of Murex based on the information in this repository and `murex.rocks`, and came up with the following design and color profile for the logo:

#### Design & Color Profile

```
Text: murex
Slogan: a modern shell for the rest of us

Design language: 
- Simple, in service of the keyword “modern”
- Sparkles like a magic wand, in service of the keywords "intuitive", "content-aware"

Colors: Calm blue to a hint of purple

Detail:
- The main sparkle should have its vertical line longer, resembling an abstraction of a Murex
- Asymmetrical; Murex is a community project, not a business. It should be fun and cool
- Clean lines avoiding overly decorative elements
- Pleasant rounded edges
- Reliance on gradients and glow to complement the sparkles
- The gradient and sparkles volume should lean on the side of subtlety to avoid overwhelming the design

Font: 
- Sans-serif, potentially slab-serif typeface
- Even weight distribution
- Lowercase, no punctuations; reflecting a modern, fast, and terminal-like style
```


After some variations and experimentations based on the above profile, I settled on a design including the [FiraCode font](https://github.com/tonsky/FiraCode) which is [licensed under the SIL Open Font License 1.1](https://github.com/tonsky/FiraCode/blob/master/LICENSE), permitting private & commercial use & distribution. This font felt right since it is a highly popular choice for terminals.

I've also added some flair for the slogan, inspired by shell instructions commonly seen in Getting Started docs;

Here is the result:

### murex.jpg
![murex](https://github.com/lmorg/murex/assets/2014360/52da962a-466a-437c-a5a2-798c9af00673)

@lmorg Please let me know your thoughts -- if you think it's the right fit, I will provide variations, like one without glow and with a transparent background that can be used as a favicon or navbar icon. Please let me know if you have specific ones in mind.

I should also credit [@aruaycodes](https://github.com/aruaycodes) (my girlfriend) for helping with the design and providing sanity checks.

**Reply by lmorg** [reply]
Thank you so much for this @yamcodes !

It's definitely better than the current logo but there's a few concerns I have. The biggest being how transparency would work.

I'm not a particular fan of the type-face used in "murex", plus it kerning feels off. Where the first half of the name feels squashed but the second half is stretched (in comparison to the first half).

As for the tag line, I'd probably leave that off as it's something that is more part of the overall design spec rather than the logo itself. But if it is kept, I'd prefer if it didn't look like a Bourne Shell code block because Murex's syntax is quite different to sh/dash/bash/etc. So having a `$` prompt with `sh` in superscript is a little deceptive.

The logo itself is pretty good though. I like how the main star is meant to symbolise a murex shell. Clever idea. Do you think it would work better rotated 90 degrees, so that the longer sparkles are going horizontally rather than vertically? My thinking being that shell windows (and monitors in general) are usually of landscape orientation. Maybe even have `»_` inside the sparkle itself? (not suggesting any of these are good ideas though :laughing:)

One other thought, I've played around with gradients on early attempts of logos I've tried to do. And I almost always preferred the logos with a solid fill instead. Gradients can looks rather busy (a point you made yourself in your brief). In fact when you look at logos for other projects, companies, and brands, they almost never feature gradients. So I wonder if the "murex" text should be outlined with a solid fill?

On the whole, I do really like what you've done there. Thank you so much for your time.

**Comment by tiymat** [comment]
I have 0 artistic skill (see below), but maybe the `Murex` name could be incorporated into the image of the shell by using the spines to make up the letters like this?
![murex_sketch](https://github.com/lmorg/murex/assets/138939221/5994b2a7-a70b-4bfd-9fa6-d5f2f716b409)

**Reply by lmorg** [reply]
That's clever :)
